scripts/cnv_wdl/somatic/cnv_somatic_pair_workflow.wdl

```diff
@@ -0,0 +1,112 @@
+# Workflow for running GATK CNV (and optionally, ACNV) on tumor/normal or tumor-only cases. Supports both WGS and WES.
+#
+# Notes:
+#
+# - The target file (targets) is required for the WES workflow and should be a TSV file with the column headers:
+#    contig    start    stop    name
+#   These targets will be padded on both sides by the amount specified by PadTargets.padding (default 250).
+#
+# - If a target file is not provided, then the WGS workflow will be run instead and the specified value of
+#   wgs_bin_size (default 10000) will be used.
```

Comment:
You can set it as a task-level parameter to CollectCoverage in the json.  Note that you will have to do this separately for both the tumor and the normal.  I'll add this to the readme as an example.